source/elements/oneDAL/source/common_interface/header_files_and_namespaces.rst

```diff
@@ -1,3 +1,61 @@
-===========================
-Header files and namespaces
-===========================
+============
+Header files
+============
+
+oneDAL public identifiers are represented in the following header files:
+
+.. list-table::
+   :widths:  10 50
+   :header-rows: 1
+
+   * - header file
+     - description
+   * - ``onedal/onedal.hpp```
+     - Main header file of oneDAL library.
```

Comment:
```suggestion
     - The main header file of oneDAL library.
```

do you need say what it contains?

Reply:
Originally it was something like "including all public headers of oneDAL library" but it was decided to make it shorter.